# DP: Description: Make glibc-2.3.5 compile with gcc-4.0 on 64bit archs.
# DP: Related bugs: 
# DP: Dpatch author: GOTO Masanori <gotom@debian.org>
# DP: Patch author: Jakub Jelinek <jakub@redhat.com>
# DP: Upstream status: Debian-Specific
# DP: Status Details: 
# DP: Date: 2005-07-14

https://www.redhat.com/archives/fedora-cvs-commits/2005-March/msg00408.html

	* sysdeps/wordsize-64/strtol_l.c: Don't add aliases if UNSIGNED.
	* sysdeps/wordsize-64/wcstol_l.c: Likewise.

===================================================================
RCS file: /cvs/glibc/libc/sysdeps/wordsize-64/strtol_l.c,v
retrieving revision 1.4
retrieving revision 1.4.2.1
Index: b/sysdeps/wordsize-64/strtol_l.c
===================================================================
--- a/sysdeps/wordsize-64/strtol_l.c	2008-08-08 19:08:18.000000000 +0200
+++ b/sysdeps/wordsize-64/strtol_l.c	2008-08-08 19:09:04.000000000 +0200
@@ -8,7 +8,9 @@
 #undef ____strtoll_l_internal
 #undef __strtoll_l
 #undef strtoll_l
+#if !UNSIGNED
 strong_alias (____strtol_l_internal, ____strtoll_l_internal)
 libc_hidden_ver (____strtol_l_internal, ____strtoll_l_internal)
 weak_alias (__strtol_l, __strtoll_l)
 weak_alias (__strtol_l, strtoll_l)
+#endif
Index: b/sysdeps/wordsize-64/wcstol_l.c
===================================================================
--- a/sysdeps/wordsize-64/wcstol_l.c	2008-08-08 19:08:18.000000000 +0200
+++ b/sysdeps/wordsize-64/wcstol_l.c	2008-08-08 19:09:04.000000000 +0200
@@ -8,6 +8,8 @@
 #undef ____wcstoll_l_internal
 #undef __wcstoll_l
 #undef wcstoll_l
+#if !UNSIGNED
 strong_alias (____wcstol_l_internal, ____wcstoll_l_internal)
 weak_alias (__wcstol_l, __wcstoll_l)
 weak_alias (__wcstol_l, wcstoll_l)
+#endif
